### DISCUSSION TITLE
Router rediscovery on HA reboot

### DISCUSSION THREAD

**Discussion by elliot76** [Vaskivskyi/ha-asusrouter#580] [Q&A]
Hey,

Since 0.21.1 everytime i reboot HA the asus router is automatically rediscovered even though it is in the devices already. 

I am running Home Assistant 2023.3.4 in a VM on a synology nas.
It doesnt prevent the already configured version from running.

Many thanks

**Comment by swin66** [comment]
I have this problem also, and it's created 4 instances of the connection under the integration and I can't get rid of them or the integration. I'm on Raspberry Pi, latest versions of everything

**Comment by Vaskivskyi** [comment]
Hello. Please, follow the suggestion in https://github.com/Vaskivskyi/ha-asusrouter/issues/581#issuecomment-1493038960 and let me know of your findings. I will close this discussion, since the issue in the link is the best place to discuss the problem you have
